installer.sh

```diff
@@ -0,0 +1,413 @@
+#!/bin/bash
+# Platform.sh CLI installer.
+
+set -euo pipefail
+
+# Location of install log
+: "${PLATFORMSH_CLI_INSTALL_LOG:=/tmp/platformsh-install-$(date '+%Y%m%d-%H%M%S')}.log}"
+
+# Define this to force install method
+: "${PLATFORMSH_CLI_INSTALL_METHOD:=}"
+
+: "${PLATFORMSH_CLI_URL:=https://github.com/platformsh/cli/releases/download}"
+
+# Force Platform.sh CLI installation in this directory instead of system directory
+: "${PLATFORMSH_CLI_INSTALL_DIR:=}"
+
+# Platform.sh CLI version to install
+: "${PLATFORMSH_CLI_VERSION:=}"
+
+# macOS specifics
+: "${PLATFORMSH_CLI_BREW_TAP:=platformsh/tap}"
+: "${PLATFORMSH_CLI_BREW_FORMULA:=platformsh/tap/platformsh-cli}"
+
+# global variables
+binary="platform"
+cmd_download=""
+cmd_shasum=""
+cmd_sudo=""
+dir_bin="/usr/bin"
```

Comment:
IMHO it would be nice to default to a user-local install not system-wide, so root isn't needed, i.e. defaulting to `~/.local/bin` where that is in the PATH?

Reply:
Great idea, fixed! See https://github.com/platformsh/cli/pull/38#discussion_r1088995989